mklive.sh

```diff
@@ -153,10 +154,12 @@ ignore_packages() {
 enable_services() {
     SERVICE_LIST="$*"
     for service in $SERVICE_LIST; do
-        if ! [ -e $ROOTFS/etc/sv/$service ]; then
-            die "service $service not in /etc/sv"
+        if ! [[ $IGNORE_SV =~ (^|[[:space:]])$service($|[[:space:]]) ]] ; then
```

Comment:
as of #384 i've started moving these stringy arrays to bash arrays, would be nice to do here too.

imo this is the most concise way of checking for array membership: https://github.com/void-linux/void-packages/pull/42656/files#diff-2aed335c529e5e6ba673dd15b329eba3605f22c2762850121093db28c4fa2f93R325-R328